contextproject-ms3/src/test/java/contextproject/loaders/FolderLoaderTest.java

```diff
@@ -0,0 +1,76 @@
+package contextproject.loaders;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+
+public class FolderLoaderTest {
+
+  ArrayList<String> array;
+  private static String directory;
+
+  /**
+   * Before running the test this method is done first to initialize the loader.
+   */
+  @Before
+  public void initialize() {
+    directory = System.getProperty("user.dir").toString() + "\\src\\test\\resources\\loadTest";
```

Comment:
This is window-specific stuff. A better idea would be to use ```File.separator``` instead of the \\'s